app/Http/Controllers/Internlink/InternlinkController.php

```diff
@@ -110,7 +110,7 @@ public function saveInternship(Request $request)
     {
         $form = $this->form(InternshipForm::class);
         $form->validate(Internship::rules(), [
-            'dateformat' => 'Date should be a valid date of the format YYYY-MM',
+            'dateformat' => 'Date should be a valid date of the format YYYY-MM-DD',
```

Comment:
I noticed that previously the date was being saved in mysql as e.g YYYY-MM-01 and I thought it would be a little weird to have the day wrong. I also couldn't figure out how to save partial dates. But I guess its OK because the day will never be displayed.